### PR TITLE
SASSify

### LINKED ISSUE #62: Migrate to SASS
Cause CSS is the suuuck

### PR DESCRIPTION
**Pull request by taylortrimble**
I have a present for you @mplewis!

Closes #62.

### CODE REVIEW

File: sass/_border-shadow.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@mixin border-shadow($depth, $color) {
```
Comment (mplewis) [inline sass/_border-shadow.scss:1]: UNFFFF MIXINS

File: .gitignore [hunk 2]
```diff
@@ -17,3 +17,4 @@ pickle-email-*.html
 .DS_Store
 *.pyc
 config.py
+static/css
```
Comment (mplewis) [inline .gitignore:20]: LOL I love this. "Fuck CSS!"

**Comment by mplewis** [comment]
Awesome dude! Is there any more work to be done?

Also, I take it you like SCSS more than SASS (Python-style indents)? I like SASS personally but it matters not to me.

File: .gitignore (hunk 2 above)
Comment (taylortrimble) [inline .gitignore:20]: I know right!? :smile:

**Comment by taylortrimble** [comment]
Nope, this is it! Works and ready to go now.

I actually like SASS better as well, but SCSS is the primary documented and supported and discussed syntax now, so I just went with it. I do think that's probably the way to do it.

**Pull request merged**